ScrapeRepos.ts

```diff
@@ -0,0 +1,356 @@
+const axios = require('axios');
+const cheerio = require('cheerio');
+const time = require('console');
+const showdown = require('showdown');
+
+
+/** Fetches the repo owner and name for each repo on the GitHub trending page
+ * @param {number} timeMode 0: daily; 1: weekly; 2: monthly
+ * @returns {string[]} an array that stores alternatingly the owner and the name of each repo
+ */
+async function fetchRepos( timeMode: number) {
```

Comment:
Please beatify your code

Reply:
Do you mean just rewriting the description as I did here or what do you mean?

Reply:
I mean reformat the code. For example there is an extra space between parantheses and timeMode. It should be like fetchRepos(timeMode: number)